### ISSUE TITLE
Contrast needs to be higher

### ISSUE THREAD

**Issue by LukasHirt** [owncloud/owncloud-design-system#444]
## Description
Contrasts are too low: WCAG 2.1 AA needs 4.5 to 1, the Phoenix UI has lower: 3.85 to 1 up to 4.09 to 1

**Comment by wuenschedesign** [comment]
The new drafts in the different issues are meeting contrast requirements.

**Comment by pascalwengerter** [comment]
Should be fixed thanks to @fschade's efforts in #1302 and #1316
